backend/routes.py

```diff
@@ -0,0 +1,39 @@
+from flask import Flask, jsonify, request
+import os
+
+app = Flask(__name__)
+
+#dummy data
+sentences = [{'sass' : 'sample_response_1'}, {'sass' : 'sample_response_2'}, {'sass' : 'sample_response_3'}]
+
+#main route defaults
+@app.route('/', methods = ['GET'])
+def main_route():
+        return jsonify({"about":"Default"})
+
+#endpoint GET ALL (UNUSED)
+@app.route('/sentences', methods = ['GET'])
+def return_All():
+    return jsonify({'sentences' : sentences})
+
+#endpoint GET SPECIFIC (UNUSED)
+@app.route('/sentences/<string:sass>', methods = ['GET'])
+def return_EndPoint(sass):
+    sens = [sentences for sentences in sentences if sentences['sass'] == sass]
+    return jsonify({'sentences' : sens[0]})
```

Comment:
This gives an error.  `IndexError: list index out of range`
 I think setting sens is done incorrectly.

Reply:
yea, reused the sentences variable in dummy data and the index